code/modules/reagents/reagent_containers/food/mobcube.dm

```diff
@@ -0,0 +1,115 @@
+/obj/item/weapon/reagent_containers/food/snacks/monkeycube
+	name = "monkey cube"
+	desc = "Just add water!"
+	icon_state = "monkeycube"
+	bitesize = 12
+	food_flags = FOOD_MEAT
+	var/mob/living/carbon/contained_mob = new /mob/living/carbon/monkey(null) //Storing it in nullspace is fine
```

Comment:
Why are you immediately initializing all monkey cubes. This is just inefficient.